src/screens/DetailGroupImage/index.js

```diff
@@ -53,13 +52,8 @@ const ShowDetailGroupImage = (props) => {
         data={images}
         initialScrollIndex={indexScroll}
         renderItem={({item, index}) => (
-          <TouchableOpacity
-            key={index}
-            onPress={() => openDetail(item.asset_url)}>
-            <ImageBackground
-              resizeMode="cover"
-              style={styles.image}
-              source={{uri: item.asset_url}}>
+          <TouchableOpacity key={index} onPress={() => openDetail(item.asset_url)}>
+            <ImageBackground resizeMode="cover" style={styles.image} source={{uri: item.asset_url}}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The formatting of the `TouchableOpacity` component has been changed. While this doesn't affect the functionality, it does impact readability. It's generally a good practice to keep the props on separate lines for better readability, especially when there are multiple props or nested components.
```diff
- <TouchableOpacity key={index} onPress={() => openDetail(item.asset_url)}>
+ <TouchableOpacity
+   key={index}
+   onPress={() => openDetail(item.asset_url)}
+ >
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->